src/lib.rs

```diff
@@ -73,7 +73,7 @@ mod world;
 
 pub use archetype::{Archetype, ArchetypeColumn};
 pub use batch::{BatchIncomplete, BatchWriter, ColumnBatch, ColumnBatchBuilder, ColumnBatchType};
-pub use bundle::{Bundle, DynamicBundle, MissingComponent};
+pub use bundle::{Bundle, DynamicBundle, DynamicBundleClone, DynamicClone, MissingComponent};
```

Comment:
Since `DynamicClone` is only useful for implementing `#[doc(hidden)]` interfaces ,its reexport should itself be `#[doc(hidden)]`, in the commented block below for that purpose.